components/map/MapForPrint.vue

```diff
@@ -1,233 +1,450 @@
                     </div>
-                </l-control>
-
-                <l-control-scale
-                    v-if="user.settings.map_scale_visible"
-                    position="bottomleft"
-                />
-
-                <l-geo-json
-                    ref="interestArea"
-                    :geojson="interestArea"
-                    :options-style="interestStyle"
-                    :visible="showInterestArea"
-                />
-
-                <SupportLayers />
-
-                <MonitoringLayers v-if="!monitoringGeoserver" :map="map" />
-
-                <BaseWmsMetadataPopup :map="map" />
-
-                <PriorityLayers :map="map" />
-            </l-map>
-
-            <div v-if="loading" class="loading-background">
+                </div>
+                <div class="text-center font-weight-bold">
+                    <p class="font-weight-bold mb-1">
+                        {{ titleMap }}
+                    </p>
+                    <p class="mb-2">
+                        Centro de Monitoramento Remoto - Fundação Nacional do
+                        Índio
+                    </p>
+                </div>
+                <div
+                    class="d-flex justify-center"
+                    style="height: 150px; max-height: 150px; width: 100%"
+                >
+                    <client-only>
+                        <l-map
+                            ref="miniPrintMap"
+                            id="miniPrintMap"
+                            :zoom="zoomMiniMap"
+                            :options="optionsMiniMap"
+                        >
+                            <l-tile-layer
+                                url="http://{s}.tile.osm.org/{z}/{x}/{y}.png"
+                                :attribution="attribution"
+                            ></l-tile-layer>
+                            <l-control position="topleft" class="ma-0 pa-0">
+                                <p class="ma-1 pa-0 print-mini-map-text">
+                                    LOCALIZAÇÃO DA ÁREA
+                                </p>
+                            </l-control>
+                        </l-map>
+                    </client-only>
+                </div>
                 <div>
-                    <v-skeleton-loader
-                        type="chip"
-                        width="32"
-                    ></v-skeleton-loader>
-                    <v-skeleton-loader
-                        type="chip"
-                        width="32"
-                    ></v-skeleton-loader>
-                    <v-skeleton-loader
-                        type="chip"
-                        width="32"
-                    ></v-skeleton-loader>
+                    <p class="d-block ma-1">Legenda:</p>
+                    <div class="ma-1 flex-wrap" style="width: 100%">
+                        <div v-if="showFeatures">
+                            <v-row no-gutters align="center">
```

Comment:
My suggestion is to iterate over data to create these legends.

Reply:
I agree, my suggestion is to keep it this way to avoid refactoring because it will take time, this information will be returned from the back in the future, requiring intervention in the code anyway, at that moment the iteration will be done correctly, what do you think? if you prefer I can do it.